app/renderer/js/main.js

```diff
@@ -153,6 +162,76 @@ class ServerManagerView {
 		this.toggleSidebar(showSidebar);
 	}
 
+	async queueDomain(domain) {
+		try {
+			const serverConf = await DomainUtil.checkDomain(domain);
+			await DomainUtil.addDomain(serverConf);
+			return true;
+		} catch (err) {
+			logger.error(err);
+			logger.error('Could not add ' + domain + '. Please contact your system administrator.');
+			return false;
+		}
+	}
+
+	async initPresetOrgs() {
+		const preAddedDomains = DomainUtil.getDomains();
+		this.presetOrgs = EnterpriseUtil.getConfigItem('presetOrganizations', []);
+
+		// if isAdminOnly is set, then no organizations not in enterprise config should be allowed
+		if (EnterpriseUtil.isAdminOnly('presetOrganizations')) {
+			for (const domain in preAddedDomains) {
+				if (this.presetOrgs.indexOf(preAddedDomains[domain].url) === -1) {
+					// domain violates enterprise config
+					DomainUtil.removeDomain(domain, true);
+				}
+			}
+		}
+
+		// set to true if at least one new domain is added
+		const domainPromises = [];
+		for (const url in this.presetOrgs) {
+			if (DomainUtil.duplicateDomain(this.presetOrgs[url])) {
+				continue;
+			}
+			domainPromises.push(this.queueDomain(this.presetOrgs[url]));
+		}
+		const domainsAdded = await Promise.all(domainPromises);
+		if (domainsAdded.includes(true)) {
+			// at least one domain was resolved
+			if (preAddedDomains.length > 0) {
+				// user already has servers added
+				// ask them before reloading the app
+				dialog.showMessageBox({
+					type: 'question',
+					buttons: ['Yes', 'Later'],
+					defaultId: 0,
+					message: 'New server' + (domainsAdded > 1 ? 's' : '') + ' added. Reload app now?'
+				}, response => {
+					if (response === 0) {
+						ipcRenderer.send('reload-full-app');
+					}
+				});
+			} else {
+				ipcRenderer.send('reload-full-app');
+			}
+		} else {
+			// no enterprise domain added
+			const { title, content } = Messages.failEnterpriseOrgError(this.presetOrgs.length);
```

Comment:
```suggestion
			const { title, content } = Messages.enterpriseOrgError(this.presetOrgs.length);
```

This should be `enterpriseOrgError` since that was added to `Message` in this commit?